docs/preview/features/secret-store/azure-functions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Using secret store within Azure Functions"
+layout: default
+---
+
+# Using secret store within Azure Functions
+
+## Installation
+For this feature, the following package needs to be installed:
+
+```shell
+PM > Install-Package Arcus.Security.AzureFunctions
+```
+
+## Usage
+The secret stores are configured during the initial application build-up in the `Startup.cs`:
+
+```csharp
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+
+[assembly: FunctionsStartup(typeof(Startup))]
+
+namespace MyHttpAzureFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.ConfigureSecretStore(stores =>
+            {
+                stores.AddEnvironmentVariables();
```

Comment:
I don't think we need this here, as this is also added on line 34

Reply:
Ah yes, that's correct. Let's remove this.